src/models/network.js

```diff
@@ -114,6 +114,17 @@ Network.prototype.validate = function (client) {
 	this.proxyPassword = cleanString(this.proxyPassword);
 	this.proxyEnabled = !!this.proxyEnabled;
 
+	const error = function (network, text) {
+		network.channels[0].pushMessage(
```

Comment:
Are you against removing `network` and just using `this.channels`?

Reply:
iirc that didn't work for some reason, I think `this` gets bound to something else